packages/phrases/src/locales/ko-kr/errors.ts

```diff
@@ -61,16 +61,18 @@ const errors = {
     general: '연동 중에 알 수 없는 오류가 발생했어요. {{errorDescription}}',
     not_found: '{{type}} 값을 가진 연동 종류를 찾을 수 없어요.',
     not_enabled: '연동이 활성화 되지 않았어요.',
+    invalid_metadata: "The connector's metadata is invalid.", // UNTRANSLATED: need to check KO_KR translation
```

Comment:
- just `// UNTRANSLATED` would be fine
- or use `ko_KR` for a more precise expression